x-pack/plugins/alerting/server/task_runner/task_runner.test.ts

```diff
@@ -362,104 +334,6 @@ describe('Task Runner', () => {
     ).toHaveBeenCalled();
   });
 
-  test.each(ephemeralTestParams)(
-    'actionsPlugin.execute is called per alert alert that is scheduled %s',
```

Comment:
i think we want to keep all of these tests, just not the iteration over the different bulk enqueue execution functions. We want to keep testing that `actionsClient.bulkEnqueueExecution` is called in these scenarios.